shipyard-controller/handler/shipyardcontroller.go

```diff
@@ -472,9 +451,9 @@ func (sc *shipyardController) handleTriggeredEvent(event models.Event) error {
 }
 
 func (sc *shipyardController) proceedTaskSequence(eventScope *keptnv2.EventData, taskSequence *keptnv2.Sequence, event models.Event, shipyard *keptnv2.Shipyard, previousFinishedEvents []interface{}, previousTask string) error {
-	task, err := sc.getNextTaskOfSequence(taskSequence, previousTask)
+	task, err := sc.getNextTaskOfSequence(taskSequence, previousTask, eventScope)
```

Comment:
can we rename the `eventScope` variable to just `eventData` in the whole file?
Also rename the `getEventScope` func to `getEventData`

Reply:
as agreed, I have added some code comments about why it is namend `eventScope`. Naming this variable `eventScope` made more sense to me personally, because it was used to determine the current state, as well as the next action within a task sequence execution.